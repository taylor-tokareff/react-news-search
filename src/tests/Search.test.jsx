/**
 * @jest-environment jsdom
 */

import React from 'react';
import { render } from '@testing-library/react';
import SearchControls from '../components/controls/Search';

describe('SearchControls component', () => {
  it('renders SearchControls', () => {
    const { asFragment } = render(
      <SearchControls text="dog"
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
