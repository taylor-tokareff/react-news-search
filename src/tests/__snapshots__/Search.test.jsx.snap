// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchControls component renders SearchControls 1`] = `
<DocumentFragment>
  <label
    for="query"
  >
    Search for article here
  </label>
  <input
    id="query"
    name="query"
    placeholder="Search Here"
    type="text"
    value=""
  />
</DocumentFragment>
`;
